src/server/services/country-codes/index.ts

```diff
@@ -0,0 +1,33 @@
+import { logger } from "../logger";
+import countriesData from "./country-codes-iso-3166.json";
+
+export default function getCountryCodeFromCountryName(
+  countryName: string
+): string {
+  let selectedCountryCode = "";
+
+  if (!countryName) {
+    logger.error("getCountryCodeFromCountryName: Country name not found");
+  }
+
+  for (const [countryDataCode, countryDataName] of Object.entries(
+    countriesData
+  )) {
+    let typedCountryName = countryDataName as string;
+
+    if (Array.isArray(countryDataName)) {
```

Comment:
with the revised implementation to use the existing countries list in metadata.ts you shouldn't need this check for Array.  As an alternative implementation, you may want to look at components/lists/helpers.getCountryLawyerRedirectLink() which does something similar in a bit more concise way

Reply:
yep, I'm in favour of that approach for new/fad libraries but for ones that are battle hardened like lodash I'm fine using them.